lib/OpenQA/Command.pm

```diff
@@ -89,6 +90,13 @@ sub parse_params {
         push @{$params{$1}}, $2;
     }
 
+    return \%params unless (defined $params{template});
+
+    foreach my $tval (@{$params{template}}) {
+        my $t = Mojo::URL->new($tval);
+        $tval = Mojo::File->new($t->path)->slurp if ($t->scheme eq 'file');
```

Comment:
`scheme` might not be defined so better use `// ''` to avoid a warning:
```suggestion
        $tval = Mojo::File->new($t->path)->slurp if ($t->scheme // '') eq 'file';
```
The same counts for the other occurrence.

Reply:
Done.